packages/rich-text-editor/src/EditableRichTextContent/EditableRichTextContent.tsx

```diff
@@ -12,6 +12,7 @@ export interface EditableRichTextContentProps
   content: EditorContentArray
   labelText: string
   isLabelHidden?: boolean
+  contentHTMLAttributes?: HTMLAttributes<HTMLDivElement>
```

Comment:
```suggestion
  contentProps?: Omit<RichTextContentProps, "contents">```

We want to make this as generic as possible, utilising the `RichTextContentProps` from the original `RichTextContent` component so that any of the props can be passed (including your aria attributes!)

Reply:
Thanks @gyfchong. I have updated the PR.